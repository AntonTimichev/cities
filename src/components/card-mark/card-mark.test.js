import React from 'react';
import renderer from 'react-test-renderer';

import CardMark from './card-mark.jsx';


it(`CardMark is rendered correctly`, () => {
  const tree = renderer.create(<CardMark />).toJSON();

  expect(tree).toMatchSnapshot();
});
