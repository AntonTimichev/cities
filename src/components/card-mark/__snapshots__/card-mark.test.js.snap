// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardMark is rendered correctly 1`] = `
<div
  className="place-card__mark"
>
  <span>
    Premium
  </span>
</div>
`;
